tensorflow/core/common_runtime/mkl_cpu_allocator.h

```diff
@@ -48,8 +52,38 @@ class MklCPUAllocator : public Allocator {
 
   MklCPUAllocator() {
     VLOG(2) << "MklCPUAllocator: In MklCPUAllocator";
-    allocator_ =
-        new BFCAllocator(new MklSubAllocator, kMaxMemSize, kAllowGrowth, kName);
+
```

Comment:
Please add a test for the new functionality.

Reply:
will do.